### PR TITLE
Add helper to convert a sentence to lowercase

### PR DESCRIPTION
**Pull request by r0qs**
Add toLower helper to convert a whole sentence to lowercase.

### CODE REVIEW

**Comment by emersonborges** [comment]
OK

File: spec/helpers/toLower_spec.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+'use strict'
+
+const dust = require('../../lib/index')
+
+fdescribe('@toLower', () => {
```
Comment (emersonborges) [inline spec/helpers/toLower_spec.js]: fdescribe

**Comment by r0qs** [comment]
zupbot ready-for-qa

**Comment by emersonborges** [comment]
zupbot ready-for-qa

**Pull request merged**